x-pack/filebeat/module/gsuite/admin/config/pipeline.js

```diff
@@ -451,6 +447,29 @@ var login = (function () {
         evt.Delete("gsuite.admin.WHITELISTED_GROUPS");
     };
 
+    var parseDate = function(field, targetField) {
+        return function(evt) {
+            new processor.Timestamp({
+                field: field,
+                target_field: targetField,
+                timezone: "UTC",
+                layouts: [
+                    "2006-01-02T15:04:05Z",
+                    "2006-01-02T15:04:05.999Z",
+                    "2006/01/02 15:04:05 UTC",
+                ],
+                tests: [
+                    "2020-02-05T18:19:23Z",
+                    "2020-02-05T18:19:23.599Z",
+                    "2020/07/28 04:59:59 UTC",
+                ],
+                ignore_missing: true,
+            }).Run(evt);
+
+            evt.Delete(field);
+        };
+    };
```

Comment:
Nit:

This pattern is allocating a new Timestamp processor for each event it processes. I think it would make more sense to create a Chain in the outer function.